docs/results.md

```diff
@@ -29,6 +29,22 @@ Version   | Approach    | Micro f1 | Description
 2020.06.1 | ensemble of tfidf and scibert | 0.75 | same as 2020.06.0 with threshold 0.3
 2020.06.2 | tfidf-adaboost | 0.63 | adaboost after tfidf
 2020.06.3 | tfidf-gboost | 0.66 | same as 2020.06.2 but with gradient boosting instead
+2020.10.0 | tfidf-svm | 0.72 | same as 2020.05.2 with train data including lay summary
+2020.10.1 | tfidf-svm | 0.72 | same as 2020.05.2 with train data including lay summary and research question
+2020.10.2 | tfidf-svm | 0.69 | same as 2020.05.2 with train data including lay summary but testing on title+synopsis
+2020.10.3 | tfidf-svm | 0.69 | same as 2020.05.2 with train data including lay summary and research question but testing on title+synopsis
+2020.10.4 | tfidf-svm | 0.52 | same as 2020.05.2 but testing on titles only
```

Comment:
I don't know how much it really matters since you have the configs but these descriptions are a bit confusing and inconsistent (i.e. here it doesnt say it was trained on title+synopsis+lay summary) , and i think they should just all follow the same format, e.g. like what you have for 2020.10.8. 'train on.... test on ... '

Reply:
what i meant was that the train is the same as 2020.05.2 but i can make it as you say if it easier to read 👍

Reply:
313416f